crates/codegen/runtime/generator/src/lib.rs

```diff
@@ -79,16 +103,16 @@ struct RuntimeModel {
 }
 
 impl RuntimeModel {
-    pub fn from_language(language: &Rc<Language>) -> Result<Self> {
+    pub fn from_language(language: &LanguageModel) -> Result<Self> {
```

Comment:
I'm a bit confused about the newly added model, that just unwraps to the existing model. Additionally, built-ins are only used/generated in a single runtime (solidity rust) and ignored everywhere else (stubs, testlang, other solidity outputs).

Instead of the changes in the generator crate here, WDYT of just rendering/writing the built ins in the solidity crate directly? as this is all very solidity specific, and cannot be applied to any other runtime.

Reply:
This was an attempt at remaining generic for any language defined in Slang, so that all of them can potentially define built-ins. But since we've decided to take that apart and have the user explicitly add the built-ins all this can be re-worked.

I'll rework the built-ins generation code.